app/models/channels.js

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+export class Channels {
+    static schema = {
+        name: 'Channels',
```

Comment:
Shouldn't the name of each schema be singular based on the Realm docs? It looks like they name all their example schemas are set up like
```
export class Channel {
    static schema = {
        name: 'Channel',
    }
}
```

Reply:
I was following our schema names based on the database and what we had in redux before, but I'm going to change them